packages/flame/lib/src/components/mixins/has_paint.dart

```diff
@@ -0,0 +1,11 @@
+import 'dart:ui';
+
+import '../../../components.dart';
+import '../../palette.dart';
+
+mixin HasPaint on BaseComponent {
+  /// Use this to change the paint used (to apply tint or opacity).
+  ///
+  /// If not provided the default is full white (no tint).
+  Paint paint = BasicPalette.white.paint();
```

Comment:
The very name of the property is not clear on what it is about. Paint of what? the whole component? Who uses it? etc

Reply:
Isn't that the point? So that it can be used for anything.

Reply:
Yeah, I am with Lukas on this one, this mixin just says that a component has a paint and add a paint property like the names says

Reply:
yeah I agree, HasPaint makes it clear that it has a paint, what it is used for has to be specified by each component that uses the mixin.